spec/presenters/sufia/trophy_presenter_spec.rb

```diff
@@ -16,7 +16,7 @@
   end
 
   let(:presenter) { described_class.new(solr_document) }
-  let(:solr_document) { SolrDocument.new(id: '123456', has_model_ssim: 'GenericWork') }
+  let(:solr_document) { SolrDocument.new(id: '123456', has_model_ssim: 'GenericWork', title_tesim: 'A Title') }
```

Comment:
Aren't `has_model_ssim` and `title_tesim` multi-valued fields? Should they return arrays? I'm wondering whether this expectation sets up a mismatch between what we're testing and how the app behaves.